Sources/SwiftSyntaxBuilder/gyb_generated/BuildableCollectionNodes.swift

```diff
@@ -171,11 +171,6 @@ public struct ArrayElementList: ExpressibleByArrayLiteral, SyntaxBuildable, Expr
   }
 }
 
-extension Array: ExpressibleAsArrayElementList where Element == ExpressibleAsArrayElement {
-  public func createArrayElementList() -> ArrayElementList {
-    return ArrayElementList(self)
-  }
-}
```

Comment:
Hm, this removes the `Array: ExpressibleAsArrayElementList` conformance since `ExpressibleAsArrayElementList` now has a superprotocol (see #528 for details why generating `Array` conformances for these hierarchies is hard).

But I guess we can just use `ArrayElementList`s instead of `ArrayExpr`s in those places now.